app/lib/search/backend.dart

```diff
@@ -70,15 +71,15 @@ class SearchBackend {
         versionList.where((pv) => pv != null),
         key: (pv) => (pv as PackageVersion).package);
 
-    final indexJsonFutures = Future.wait(packages.map((p) =>
-        dartdocClient.getContentBytes(p.name, 'latest', 'index.json',
+    final pubDataFutures = Future.wait(packages.map((p) =>
+        dartdocClient.getContentBytes(p.name, 'latest', 'pub-data.json',
```

Comment:
Does pub-data.json already exist?

Reply:
Prod: not yet, but it will with the next release.